### ISSUE TITLE
[BUG] Imviz: Existing viewer not resizing properly when new viewer is added

### ISSUE THREAD

**Issue by pllim** [spacetelescope/jdaviz#1604]
**Workaround**

Resize the browser.

**Describe the bug**
Existing viewer no longer resizes properly when new viewer is created in Imviz.

**To Reproduce**
1. Open Imviz example notebook and load both the images.
2. Open a new viewer.
3. See horizonal scrollbar appear in the first viewer. Screenshot below.

![Screenshot 2022-08-29 135941](https://user-images.githubusercontent.com/2090236/187267769-34245250-327b-4c30-a1f7-510c3f70793b.jpg)

**Expected behavior**
First viewer to resize and no horizontal scrollbar would appear.

**Desktop (please complete the following information):**
 - OS: WSL2 (Debian on Windows 10)
 - Browser: Chrome Version 104.0.5112.102 (Official Build) (64-bit) -- This was installed under Windows 10
 - Jupyter: [run "jupyter --version"]

```
IPython          : 8.4.0
ipykernel        : 6.15.1
ipywidgets       : 8.0.1
jupyter_client   : 7.3.5
jupyter_core     : 4.11.1
jupyter_server   : 1.18.1
jupyterlab       : 3.4.5
nbclient         : 0.5.13
nbconvert        : 6.5.3
nbformat         : 5.4.0
notebook         : 6.4.12
qtconsole        : 5.3.1
traitlets        : 5.1.1
```

**Package versions (please complete the following information):**

```
Numpy 1.23.2
astropy 5.2.dev363+g752b961b5
matplotlib 3.5.3
scipy 1.9.0
scikit-image 0.19.3
asdf 2.13.0
gwcs 0.18.1
regions 0.6
specutils 1.8.0
specreduce 1.1.0
photutils 1.5.0
astroquery 0.4.6
pyyaml 6.0
asteval 0.9.27
idna 3.3
traitlets 5.1.1
bqplot 0.12.34
bqplot-image-gl 1.4.11
glue-core 1.5.0
glue-jupyter 0.13.0
glue-astronomy 0.5.0
echo 0.6
ipyvue 1.7.0
ipyvuetify 1.8.2
ipysplitpanes 0.2.0
ipygoldenlayout 0.4.0
Jinja2 3.1.2
voila 0.3.6
vispy 0.11.0
sidecar 0.5.1
Jdaviz 2.10.1.dev7+g8050cafc
```

p.s. I think ipypopout is 0.0.12 though it is not easy to tell given https://github.com/mariobuikhuizen/ipypopout/issues/2 and https://github.com/mariobuikhuizen/ipypopout/issues/3 .

**Additional context (e.g. data files)**

My old Python 3.9 env "suddenly" broke (i.e., I couldn't identify what got upgraded where that caused it). I remade the env from scratch and now I see this problem that I had not seen before.

**Comment by pllim** [comment]
p.s. Downgrading to "ipywidgets<8" did not help. I am a little afraid this is caused by ipygoldenlayout rotting over at https://github.com/nmearl/ipygoldenlayout

**Comment by kecnry** [comment]
Does the resize happen properly if you then manually drag to change the size of the window or is that broken too?

**Comment by pllim** [comment]
Which window? The browser window or the individual Imviz viewer?

**Comment by kecnry** [comment]
The divider between the two Imviz viewers

**Comment by pllim** [comment]
If I move the divider, it got worse (now both windows have scrollbars).

![Screenshot 2022-08-30 125249](https://user-images.githubusercontent.com/2090236/187495183-c2afb6d2-5960-413b-96b8-1a6cffcaaf9a.jpg)

However, if I resize the *browser* window, the scrollbars go away. Hmmm!

**Comment by kecnry** [comment]
Ok, as long as calling `app.vue_relayout()` manually does what you'd expect, then I would guess the `@resize="relayout"` on `splitpanes` is no longer triggering `vue_relayout` (reason unknown).

**Comment by pllim** [comment]
Calling `app.vue_relayout()` does not make scrollbars go away.

**Comment by pllim** [comment]
@camipacifici also saw a similar problem where Cubeviz displays will not resize when she opens the plugins tray.

**Comment by maartenbreddels** [comment]
noteby @pllim on call : issue on notebook only, not lab

**Comment by maartenbreddels** [comment]
For me this was an issue with jlab *and* classic notebook, and was due to
 * https://github.com/bqplot/bqplot/pull/1517 (released in 0.12.34 and 0.12.35)

Which was fixed in:
 * https://github.com/bqplot/bqplot/pull/1523 (released in 0.12.36)

So I think the fix should be to upgrade to 0.12.36. Although you did list you are using ipywidgets 8, which should not make this an issue at all, so I'm confused about that.

Please let me know if upgrading to bqplot 0.12.36 fixes the issue for you.

**Comment by pllim** [comment]
After I upgraded from bqplot 0.12.34 to 0.12.36 , the scrollbar appears briefly and then disappears when display is done resizing, so that is good enough for me. See https://github.com/spacetelescope/jdaviz/pull/1663 . Thanks for tracking this down!
